adobecom/milo
# MWPW-145513: Tabs block accessibility

## Description
This PR solves the following a11y issues for the tabs block:
1. Buttons must have discernible text
2. Elements with ARIA roles should have all required ARIA attributes

## Related Issue
Resolves: [MWPW-145513](https://jira.corp.adobe.com/browse/MWPW-145513)

## Testing instructions
1. go to the test url
2. open [axe DevTools](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)
3. scan the entire page
<img width="1546" alt="Screenshot 2024-04-17 at 10 24 02" src="https://github.com/adobecom/milo/assets/146744221/70e14fde-47e8-4dc6-b391-8cd4c585e45f">
4. you should not see any a11y errors

## Screenshots:

**Before**
![image](https://github.com/adobecom/milo/assets/146744221/cb2c5cf1-8181-4933-8981-7ff98e5130c8)

**After**
![image](https://github.com/adobecom/milo/assets/146744221/479409b0-c1f8-4ddb-be01-7f2ea3ad2dc7)


## Test URLs
**Acrobat:**
- Before: https://www.stage.adobe.com/acrobat/online/sign-pdf.html
- After: https://www.stage.adobe.com/acrobat/online/sign-pdf.html?milolibs=mwpw-145513-a11y--milo--robert-bogos

**BACOM:**
- Before: https://main--bacom--adobecom.hlx.page/products/real-time-customer-data-platform/rtcdp
- After: https://main--bacom--adobecom.hlx.page/products/real-time-customer-data-platform/rtcdp?milolibs=mwpw-145513-a11y--milo--robert-bogos

**CC:**
- Before: https://main--cc--adobecom.hlx.page/drafts/rbogos/default?martech=off
- After: https://main--cc--adobecom.hlx.page/drafts/rbogos/default?martech=off&milolibs=mwpw-145513-a11y--milo--robert-bogos

**Homepage:**
- Before: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?
- After: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?milolibs=mwpw-145513-a11y--milo--robert-bogos

**Blog:**
- Before: https://main--blog--adobecom.hlx.page/
- After: https://main--blog--adobecom.hlx.page/?milolibs=mwpw-145513-a11y--milo--robert-bogos

**Milo:**
- Before: https://main--milo--adobecom.hlx.page/drafts/rbogos/tabs-scrolling
- After: https://mwpw-145513-a11y--milo--robert-bogos.hlx.page/drafts/rbogos/tabs-scrolling
